Xamarin.Forms.Platform.iOS/Renderers/ShellPageRendererTracker.cs

```diff
@@ -284,9 +275,16 @@ async Task UpdateLeftToolbarItems()
 			if (NavigationItem.LeftBarButtonItem != null)
 			{
 				if (String.IsNullOrWhiteSpace(image?.AutomationId))
-					NavigationItem.LeftBarButtonItem.AccessibilityIdentifier = "OK";
+				{
+					if (IsRootPage)
+						NavigationItem.LeftBarButtonItem.AccessibilityIdentifier = "OK";
```

Comment:
These don't seem very global friendly 🤔

Reply:
I'm assuming "global" here refers to 🌏 and not global variables?

AFAIK this is fine because the AccessbilityLabel will be used by screen readers 

https://developer.apple.com/documentation/uikit/uiaccessibilityidentification

> You can use the identifiers you define in UI Automation scripts because the value of accessibilityIdentifier corresponds to the return value of the name method of UIAElement.

So if we want to provide translated defaults we'd use label, otherwise the buttons are just going to localize against the titles or via our other accessibility methods